scripts/e2e.oasis.sh

```diff
@@ -11,7 +11,10 @@ set -o nounset -o pipefail -o errexit
 trap "exit 1" INT
 
 # Get the root directory of the tests dir inside the repository.
-ROOT="$(cd $(dirname $0); pwd -P)"
+ROOT="$(
```

Comment:
Seems like you're using a formatter for bash scripts? Can you also update other bash scripts and add a formatter/linter commands to `fmt` and `lint` directive in `package.json`? It would also be nice to lint bash scripts on CI.

Reply:
Oh yeah i that's just prettier that i have set in my editor. Yeah i'll look into setting it up.

Reply:
done